xapi-tapctl.opam

```diff
@@ -8,6 +8,7 @@ dev-repo: "git://github.com/xapi-project/tapctl.git"
 tags: [ "org:xapi-project" ]
 build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 depends: [
+  "ocaml"
```

Comment:
I think we can remove this

Reply:
This was added by the automatic update from Opam - so I'd like to keep it. OCaml becomes more or a regular package in Opam 2 and I suspect this is the consequence.